### DISCUSSION TITLE
Fooocus on Vast.ai

### DISCUSSION THREAD

**Discussion by juliux32** [lllyasviel/Fooocus#1804] [Q&A] [answered]
Hello,

Is it possible to run Fooocus on Vast.ai ? I have found some tutorials how to run Fooocus on Runpod.io but can't find anything useful about Fooocus on Vast.ai

Have someone tried it?

**Comment by CodeWritingCow** [comment, marked as answer]
Hi @juliux32 I was able to run Fooocus on Vast.ai using a template named `Mzenko/Fooocus-Test`.

**Reply by nikolatesla20** [reply]
I don't see that template there
